### PR TITLE
Controller, Static page and unit test

### PR DESCRIPTION
**Pull request by sapphireduffy**
# Description

Please explain briefly the changes you made here.

## Checklist

- [x] 3 Amigos
- [x] Acceptance Criteria met
- [x] Branch named {feature|hotfix|task}/{S4-*}
- [x] Commit messages are meaningful
- [x] Manually tested
- [x] All unit tests passing
- [x] WAVE accessibility testing tool ran on new or updated pages
- [x] Pulled latest master into feature branch
- [x] Code reviewed
- [x] New Docker environment variables are consistent with those on the Rebel1 environment
- [x] If your pull request depends on any other, please link them in the description

https://github.com/companieshouse/cdn.ch.gov.uk/pull/419

## Screenshots of new or updated views
Screenshot 2020-07-30 at 12 24 54

### CODE REVIEW

File: src/views/certificate-signed.njk [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{% extends 'layout.njk' %}
+
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+
+{% block pageTitle %}
+  {{ serviceName }}
```
Comment (peadarkelly) [inline src/views/certificate-signed.njk:6]: Pretty sure that this should match the heading of the page. Just did a quick scan through our repo and they seem to be really inconsistent. Can you update these to match the heading of the respective pages?

File: src/views/certificate-signed.njk [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+        <h2 class="govuk-heading-s">What to do next</h2>
+
+        <p class="govuk-body">The person making the application can download a signed PDF copy of it after it has been submitted.</p>
+        <p class="govuk-body">The directors or members must then send this to all interested parties within 7 days of submission.</p>
+
+        {{ govukWarningText({
+          text: "It is a criminal offence not to tell all interested parties that an application has been made to close the company.",
+          iconFallbackText: "Warning"
+        }) }}
```
Comment (peadarkelly) [inline src/views/certificate-signed.njk:32]: @mateuszc-kainos nunjucks formatting - yay or nay?

File: src/controllers/certificateSigned.controller.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+
```
Comment (peadarkelly) [inline src/controllers/certificateSigned.controller.ts:1]: no need for a new line at the top of the file

File: src/controllers/certificateSigned.controller.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+import BaseController from 'app/controllers/base.controller'
+import { CERTIFICATE_SIGNED_URI } from 'app/paths'
+import TYPES from 'app/types'
+
+@controller(CERTIFICATE_SIGNED_URI, TYPES.SessionMiddleware, TYPES.AuthMiddleware, TYPES.CompanyAuthMiddleware)
+export class CertificateSignedController extends BaseController {
+
+  @httpGet('')
+  public async get(): Promise<string> {
+
```
Comment (peadarkelly) [inline src/controllers/certificateSigned.controller.ts:13]: I've noticed this in a few places - why the new line at the start of the method?

File: src/views/certificate-signed.njk (hunk 1 above)
Comment (sapphireduffy) [reply src/views/certificate-signed.njk:6]: Will do!

File: src/views/certificate-signed.njk (hunk 2 above)
Comment (mateuszc-kainos) [reply src/views/certificate-signed.njk:32]: Strong yay

File: test/controllers/certificateSigned.test.ts [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+     const app = createApp()
+
+     const res = await request(app)
+      .get(CERTIFICATE_SIGNED_URI)
+      .expect(OK)
+
+     const htmlAssertHelper: HtmlAssertHelper = new HtmlAssertHelper(res.text)
+
+     assert.isTrue(htmlAssertHelper.hasText('h1', 'Application signed'))
+    })
```
Comment (peadarkelly) [inline test/controllers/certificateSigned.test.ts:24]: i think you have only indented 1 space here

**Pull request merged**